app/controllers/posts_controller.rb

```diff
@@ -7,20 +7,22 @@ def index
     @posts = Post.all
   end
 
-  def show; end
+  def show
+  end
 
   def new
     @post = current_user.posts.build
   end
 
-  def edit; end
+  def edit
+  end
 
   def create
     @post = Post.create(post_params.merge(user: current_user))
 
     respond_to do |format|
       if @post.save
-        format.html { redirect_to @post, notice: I18n.t('controllers.posts.created') }
+        format.html { redirect_to @post, notice: I18n.t('activerecord.controllers.posts.created') }
```

Comment:
Translations are better to be split by domains like:
`user`, `post` etc...
```suggestion
        format.html { redirect_to @post, notice: I18n.t('post.created') }
```